README.md

```diff
@@ -130,23 +132,29 @@ See the [migration document](MIGRATION.md) for more details.
 Run the JAR file with no arguments to get the usage message:
 
 ```
+
+
 $ java -jar cromwell.jar
 java -jar cromwell.jar <action> <parameters>
 
 Actions:
-run <WDL file> [<JSON inputs file> [<JSON workflow options>
-  [<OUTPUT workflow metadata>]]]
+run <WDL file> [<JSON inputs file>] [<JSON workflow options>]
+  [<OUTPUT workflow metadata>] [<Directory of WDL Files>]
```

Comment:
Has this really changed with every parameter optional and Cromwell just figures it all out somehow?  The old spec was nested from the end to the front.